### PR TITLE
Enable logback logging configuration and support rolling log files 

### PR DESCRIPTION
**Pull request by sven1103**
Enables a sophisticated logging configuration for console + rolling files. The log path is also configurable via an environmental variable.

### CODE REVIEW

File: src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %black(%d{ISO8601}) %highlight(%-5level) [%blue(%t)] %yellow(%logger{36}): %msg%n%throwable
+      </Pattern>
+    </layout>
+  </appender>
+
+  <appender name="RollingFile"
+    class="ch.qos.logback.core.rolling.RollingFileAppender">
+    <file>${LOGS}/datamanager-logger.log</file>
```
Comment (Steffengreiner) [inline src/main/resources/logback-spring.xml:17]: I think the log name here is a copy paste oversight since this is not the datamanager? 🤔

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Heya @sven1103! Nicely done, however i'd suggest changing the name of the generated log file

File: src/main/resources/logback-spring.xml (hunk 1 above)
Comment (sven1103) [reply src/main/resources/logback-spring.xml]: well spotted

**Review by Steffengreiner — APPROVED** [review]
well done!

**Pull request merged**